packages/at_secondary_server/lib/src/verb/handler/abstract_update_verb_handler.dart

```diff
@@ -197,12 +201,41 @@ abstract class AbstractUpdateVerbHandler extends ChangeVerbHandler {
     unawaited(notificationManager.notify(atNotification));
     return atNotification;
   }
+
+  /// If metadata contains "null" string, then reset the metadata. So set it to null
+  /// If metadata contains null (null object), then fetch the existing metadata.If
+  /// existing metadata value is not null, set it the current AtMetaData obj.
+  AtMetaData _setNullOrExistingMetadata(
```

Comment:
we can rename this to _unsetOrRetainMetadata

Reply:
Sure Murali, Will rename the method